src/genericComponents/Input.tsx

```diff
@@ -153,7 +153,7 @@ const InputField = (props: any) => {
 };
 
 const CAGInput = styled(Form.Group)`
-  box-shadow: 0px 13px 21px -10px rgba(0, 0, 0, 0.3);
+  box-shadow: 0px 13px 21px -10px rgba(0, 0, 0, 0);
```

Comment:
You can remove this box-shadow line altogether!

Reply:
Just added a new commit w/ line deleted and w/o package-lock in the push